added PasteBar app

kurdin opened the pull request.

talynone commented:
This submission has a few errors in it, .DS_Store and assets/.DS_Store should not be part of the commit, these are MacOS metadata files (maybe add DS_Store to .gitignore?).

Also this erroneously replaces the "sets/tools/ambermoon.png" icon with the PasteBar icon.

kurdin commented:
@talynone sorry about the mess. I reverted `assets/tools/ambermoon.png` icon and removed `.DS_Store` files. Also I added `.gitignore` with `.DS_Store`.

Should be good now! Thank you!
